.semaphore/semaphore.yml

```diff
@@ -0,0 +1,72 @@
+version: v1.0
+name: Dockstore-UI2 Basic Test
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: "Build"
+    task:
+      env_vars:
+        - name: APP_ENV
+          value: prod
+        - name: WEBSERVICE_VERSION
+          value: 1.6.0-alpha.6
+      prologue:
+        commands:
+          - nvm use 10.10.0

```

Comment:
Just curious, why is this different than .travis.yml (10.13.0)?

Reply:
Closest version that Semaphore has

Reply:
Nevermind, was reading old docs https://semaphoreci.com/docs/languages/javascript/javascript-support-on-semaphore.html vs https://docs.semaphoreci.com/article/82-language-javascript-and-nodejs